src/raft.c

```diff
@@ -306,6 +306,26 @@ void RaftExecuteCommandArray(RedisRaftCtx *rr,
                              RaftRedisCommandArray *cmds)
 {
     RedisModuleCallReply *reply;
+    RedisModuleUser *user = NULL;
+
+    if (cmds->acl) {
+        int nokey;
+        user = RedisModule_DictGet(rr->acl_dict, cmds->acl, &nokey);
+        if (nokey) {
+            char user_name[64];
+            uint64_t count = RedisModule_DictSize(rr->acl_dict);
+            count++;
+            snprintf(user_name, 64, "redis_raft%lu", count);
+            user = RedisModule_CreateModuleUser(user_name);
+            RedisModule_Assert(user != NULL);
+
+            size_t acl_len;
+            const char *acl_str = RedisModule_StringPtrLen(cmds->acl, &acl_len);
+            RedisModule_Assert(RedisModule_SetModuleUserACLString(ctx, user, acl_str, NULL) == REDISMODULE_OK);
```

Comment:
```suggestion
            int ret = RedisModule_SetModuleUserACLString(ctx, user, acl_str, NULL);
            RedisModule_Assert(ret == REDISMODULE_OK);
```

Function call in an assert statement feels like something not important. Even though `RedisModule_Assert()` is a runtime check, traditionally, `assert()` is stripped out in the release build. So, I personally tend to ignore statements in an assert while reading the code. Let's have it in another line to help me while reading the code :)